draft-ietf-scitt-architecture.md

```diff
@@ -387,7 +387,7 @@ Registration Policies refer to additional checks over and above the Mandatory Re
 Transparency Services MUST maintain Registration Policies.
 
 Transparency Services MUST also maintain a list of trust anchors, which SHOULD be used by Relying Parties to authenticate Issuers, and which MAY be included in a Registration Policy statement.
-For instance, a trust anchor could be an X.509 root certificate, the discovery URL of an OpenID Connect identity provider, or any other COSE compatible PKI trust anchor.
+For instance, a trust anchor could be an X.509 root certificate, the URL pointing to an OpenID Connect identity provider combined with the use of the Web PKI, or a JSON Web Key Set containing the public keys of Issuers.
```

Comment:
```suggestion
For instance, a trust anchor could be an X.509 root certificate, pointing to an OpenID Connect identity provider, or any other COSE-compatible trust anchor.
```

Reply:
I would be happy to approve if this change is integrated.

Reply:
So would I

Reply:
Give me examples of "any other COSE-compatible trust anchor". 

When the sentence starts with "for instance" or "for example" it has to actually give specific examples rather than leaving the reader puzzled about what you actual mean.

I have worked on COSE for years and I don't know what you are referring. How would any other reader know what you mean?